app/assets/src/components/views/SampleUploadFlow/HostOrganismMessage.jsx

```diff
@@ -37,30 +56,72 @@ export default class HostOrganismMessage extends React.Component {
         <strong>
           {count} sample{count > 1 ? "s" : ""}
         </strong>{" "}
-        that you indicated as a <strong>{host}</strong> Host Organism,{" "}
-        {this.hasMatch(host) ? (
+        {this.renderHostPhrase(host, count)}{" "}
+        {this.hasMatch(host) && !this.isERCC(host) ? (
           <span>
-            we will subtract out a <strong>{host}</strong> genome.
+            we will subtract out reads that align to a <strong>{host}</strong>{" "}
+            genome
           </span>
         ) : (
-          <span>we will only remove ERCCs.</span>
+          <span>we will only remove ERCCs</span>
         )}
+        {host !== "Human" && " and reads that align to the human genome"}.
+      </span>
+    );
+  }
+
+  isERCC(host) {
+    return host.toLowerCase() === "ercc only";
+  }
+
+  renderHostPhrase(host, count) {
+    // special case explict choice of ERCC to avoid awkward phrasing
+    return (
+      <span>
+        {this.isERCC(host) ? (
+          <span>that you indicated {count > 1 ? "are" : "is"} ERCC Only</span>
+        ) : (
+          <span>
+            that you indicated {count > 1 ? "are" : "is"} from a{" "}
+            <strong>{host}</strong> Host Organism
+          </span>
+        )},
       </span>
     );
   }
 
   renderManyHosts(uniqHosts) {
+    // Strangely, the " " spaces are not being ending up in the header so
+    // we use &nbsp;
```

Comment:
You can also insert a string in curly braces, e.g. `{"  "}` to force React to render an exact amount of spaces or other characters, just fyi

Reply:
Yea, that's what I first tried and it strangely didn't work in this component.